examples/AeroShield/AeroShield_OpenLoop/AeroShield_OpenLoop.ino

```diff
@@ -0,0 +1,50 @@
+/*
+  This code is part of the AutomationShield hardware and software
+  ecosystem. Visit http://www.automationshield.com for more
+  details. This code is licensed under a Creative Commons
+  Attribution-NonCommercial 4.0 International License.
+
+  Created by 
+  1ast update: 13.03.2022.
+*/
+
+#include "AeroShield.h"           //  Include main library
+
+  float startangle=0;           //  Variable for storing 0° angle in raw format
```

Comment:
I think there is some kind of a problem with me uploading the stuff on GIT. I changed the camelCase variables before you corrected me! but the shitty thing is still there... Im gonna ask anka about it because I really dont know where the problem is. Also the same is about spelling milliseconds :D

Reply:
All right. I think this is the part where you find out what went wrong and learn Git in the process:)